schedule/scheduler.lua

```diff
@@ -33,6 +35,7 @@ function Scheduler:initialize(checks)
   self._checkMap = {}
   self._checks = {}
   self._runCount = 0
+  self._minCheckPeriod = 0xffff
```

Comment:
if the value stays as `0xffff` what will it actually look like in the payload? maybe it is best to start this out at the max check period(30 minutes)?